server/lib/scene/scene.actions.js

```diff
@@ -27,7 +51,17 @@ const actionsFunc = {
       device = self.stateManager.get('device', action.device);
       deviceFeature = getDeviceFeature(device, action.feature_category, action.feature_type);
     }
-    return self.device.setValue(device, deviceFeature, action.value);
+
+    let { value } = action;
+    if (action.evaluate_value) {
```

Comment:
Please avoid implicit conversion, this is sensible to exact same bug as before. See my comment: 

<img width="785" alt="Screenshot 2023-04-28 at 11 39 51" src="https://user-images.githubusercontent.com/7365207/235055602-0da3c24f-99a9-4454-bd97-ae63ef1e2ccc.png">